rllib/train.py

```diff
@@ -217,59 +112,60 @@ def patch_path(path):
 
             exp["config"]["input"] = patch_path(input_)
 
-        if not exp.get("run"):
-            parser.error("the following arguments are required: --run")
         if not exp.get("env") and not exp.get("config", {}).get("env"):
-            parser.error("the following arguments are required: --env")
-
-        if args.torch:
+            raise ValueError(
+                "You either need to provide an --env argument or pass"
+                "an `env` key with a valid environment to your `config`"
+                "argument."
+            )
+        if torch:
```

Comment:
Let's get rid of this deprecated arg and enforce `--framework torch` instead.

Reply:
done